docs/auto-generating-selectors.md

```diff
@@ -44,11 +39,11 @@ interface BearState {
   increment: () => void
 }
 
-const useStoreBase = create<BearState>((set) => ({
+const useStoreBase = create<BearState>()((set) => ({
   bears: 0,
   increase: (by) => set((state) => ({ bears: state.bears + by })),
-  increment: () => set((state) => state.increase(1)),
-}))
+  increment: () => set((state) => ({ bears: state.bears += 1 }))
```

Comment:
It doesn't matter much but let's do `+` instead of `+=` to avoid unnecessary mutation or confusions

Reply:
I removed the use of 'increase' here just to make the example more simple, but for sure can add it back!

Reply:
Should be this in that case.
```suggestion
  increment: () => set((state) => ({ bears: state.bears + 1 }))
```